modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2817,14 +2817,7 @@ export class Wallet implements IWallet {
       throw new Error('prv required to sign transactions with TSS');
     }
     try {
-      const hashAlgorithm = this.coinSpecific()?.hashAlgorithm;
-      let hash: Hash;
-      if (hashAlgorithm && hashAlgorithm.startsWith('sha')) {
-        hash = createHash(hashAlgorithm);
-      } else {
-        hash = createKeccakHash('keccak256');
-      }
-
+      const hash: Hash = this.baseCoin.getHashFunction() ?? (createKeccakHash('keccak256') as Hash);
```

Comment:
If we are using `createKeccakHash('keccak256') as Hash` as the default, why not add this to baseCoin?

Reply:
done